azurelinuxagent/common/cgroups.py

```diff
@@ -602,15 +617,16 @@ def add_to_extension_cgroup(name, pid=int(os.getpid())):
         """
         if not CGroups.enabled():
             return
-        if name == AGENT_NAME:
-            logger.warn('Extension cgroup name cannot match agent cgroup name ({0})'.format(AGENT_NAME))
+        if name == AGENT_CGROUP_NAME:
+            logger.warn('Extension cgroup name cannot match extension handler cgroup name ({0})' \
```

Comment:
add something similar to "will not track extension"

Reply:
Done.